sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java

```diff
@@ -323,6 +329,116 @@ void testAssociateNewLocationForHandleSameStateTwice() {
         assertTrue(testRunObserver.isInvalid(), "Test run should have been invalid.");
     }
 
+    @Test
+    @DisplayName("testMetricStatusManipulationMSRMTActivationStateSHTDNGood: Set ActivationState "
+            + "of all MSRMT-Metrics to SHTDN.")
+    void testMetricStatusManipulationMSRMTActivationStateSHTDNGood() throws PreconditionException {
+        final ComponentActivation startActivationState = ComponentActivation.ON;
+        final MetricCategory metricCategory = MetricCategory.MSRMT;
+        final ComponentActivation activationState = ComponentActivation.SHTDN;
+        final String metricStateHandle = "metricStateHandle";
+
+        final MdibAccess mockMdibAccess = mock(MdibAccess.class);
+        when(mockDevice.getMdibAccess()).thenReturn(mockMdibAccess);
+        final MdibEntity mdibEntity = mock(MdibEntity.class);
+
+        when(mockMdibAccess.findEntitiesByType(AbstractMetricDescriptor.class)).thenReturn(List.of(mdibEntity));
+        final AbstractMetricDescriptor metricDescriptor = mock(AbstractMetricDescriptor.class);
+        when(mdibEntity.getDescriptor(AbstractMetricDescriptor.class)).thenReturn(Optional.of(metricDescriptor));
+        when(metricDescriptor.getMetricCategory()).thenReturn(metricCategory);
+        final AbstractMetricState metricState = mock(AbstractMetricState.class);
+        when(mdibEntity.getStates(AbstractMetricState.class)).thenReturn(List.of(metricState));
+        when(metricState.getDescriptorHandle()).thenReturn(metricStateHandle);
+
+        final ManipulationPreconditions.MetricStatusManipulationMSRMTActivationStateSHTDN preconditionUnderTest =
+                new ManipulationPreconditions.MetricStatusManipulationMSRMTActivationStateSHTDN();
```

Comment:
You do not need an instance of the ManipulationPreconditions.MetricStatusManipulationMSRMTActivationStateSHTDN class because the _manipulation_ method is static

Reply:
I think this would be beneficial, currently the setMetricStatus manipualtion could fail and the test would still pass. If you use the _manipulation_ method, the return value could be used to verify the expected result. Via _verifyPrecondition_ the _manipulation_ method is called indirectly.

Reply:
I see. Done.